tests/ffi/deny_set_scheduler/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -euvx
+
+. ../common/prepare.sh
+
+local expected_result
+expected_result="Failed to set scheduler: Operation not permitted"
+
+disk_cleanup
+prepare_test
+reload_config
+
+podman exec -it qm /bin/bash -c \
```

Comment:
You can use
prepare_images()
run_container_in_qm()

Reply:
I remove "local"

Reply:
Please use this to save some redundant code 
prepare_images
# this create the argument named container running as detached
run_container_in_qm ffi-qm
#now you could continue to line_16

Reply:
Done, thank you.